xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/internal/macro/DefaultMacroManager.java

```diff
@@ -106,6 +104,9 @@
                     + "For example \"html/xwiki/2.0\". This macro will not be available in the system.");
                 continue;
             }
+            
+            Set<MacroId> result = new HashSet<MacroId>();
```

Comment:
Please build with Maven before committing. Also even if this was valid Java it would break the whole method (look at what this method is supposed to do).